app/javascript/components/pages/SongNew.js

```diff
@@ -1,11 +1,62 @@
 import React, { Component } from 'react'
+import { Form, FormGroup, Label, Input, Button } from "reactstrap";
+import { Redirect } from "react-router-dom";
 
-export class SongNew extends Component {
+class SongNew extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      form: {
+        title: "",
+        artist: "",
+      },
+      submitted: false,
+    };
+  }
+  handleChange = (e) => {
+    let { form } = this.state;
+    form[e.target.name] = e.target.value;
+    this.setState({ form: form });
+  };
+
+  handleSubmit = () => {
+    this.props.createSong(this.state.form);
+    this.setState({ submitted: true });
+  };
   render() {
-    return (
-      <div>SongNew</div>
+    console.log(this.state.form);
```

Comment:
Once you know the form is doing what you want it to do the console.logs should be removed.